Example/Tests/TestHelper.swift

```diff
@@ -30,3 +30,31 @@ class TestHelper: NSObject {
         TestHelper.shared.mockTracker.addMock(name, returnValue: returnValue)
     }
 }
+
+extension Dictionary where Key: Comparable {
+    /// Converts the dictionary to a string with keys in stable order.
+    /// - Returns: A string representation of the dictionary with sorted keys.
+    func toStringWithStableOrder() -> String {
+        let sortedKeyValuePairs = self.sorted { $0.key < $1.key }
```

Comment:
`self` is not needed here.

```suggestion
        let sortedKeyValuePairs = sorted { $0.key < $1.key }
```